src/openapi/peridio-admin-openapi.yaml

```diff
@@ -1366,6 +1366,9 @@ paths:
                   $ref: "#/components/schemas/device-last-communication"
                 tags:
                   $ref: "#/components/schemas/array-of-device-tags"
+                # Remove comment after updating API
+                # target:
+                #   $ref: "#/components/schemas/device-target"
```

Comment:
Are we expecting to delete this chunk before merging this PR?

Reply:
I think that, as I understand it, the API needs to be updated to get the documentation for the target. I could be wrong though. Andres suggested to go ahead and make a PR before the API was updated. After the update I would un comment it.